code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -203,12 +203,19 @@
 		for(var/obj/item/I in BP.embedded_objects)
 			msg += "<B>[t_He] [t_has] \a \icon[I] [I] embedded in [t_his] [BP.name]!</B>\n"
 
+	//stores how many left limbs are missing
+	var/l_limbs_missing = 0
 	for(var/t in missing)
 		if(t=="head")
 			msg += "<span class='deadsay'><B>[capitalize(t_his)] [parse_zone(t)] is missing!</B><span class='warning'>\n"
 			continue
+		if(t == "l_arm" || t == "l_leg")

```

Comment:
yeah, that's what tripped me up the first time as I had assumed someone had added defines for them, but it was not to be.